PocketKit/Sources/PocketKit/MyList/MyListTitleView.swift

```diff
@@ -88,18 +78,19 @@ class MyListTitleView: UIView {
 
         stackView.setNeedsLayout()
         stackView.layoutIfNeeded()
-        selection.frame = selectedButton.frame
     }
 
     override func layoutSubviews() {
         super.layoutSubviews()
-
-        if selection.frame == .zero,
-           let selectedButton = buttons.first(where: \.isSelected) {
-            selection.frame = selectedButton.frame
+        
+        buttons.forEach { button in
+            button.layer.cornerRadius = 16
```

Comment:
Digging through view hierarchies like this can lead to unexpected weirdness down the line if the hierarchy changes (i.e. if we add another kind of button).

I'm thinking we can configure these buttons to update the background color when their state changes by setting the `backgroundColorTransformer` on the background configuration.

```swift
class MyListSelectionButton: UIButton {
    // ...
    init(selection: MylistSelection, action: UIAction) {
        // ...
        configuration = UIButton.Configuration.plain()
        configuration?.background.backgroundColorTransformer = UIConfigurationColorTransformer { [weak self] _ in
            return isSelected ? UIColor(.ui.teal6) : .clear
        }
    }
}
```

Reply:
good point, updated!